tools/depends/target/gnutls/Makefile

```diff
@@ -31,6 +24,9 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
 	cd $(PLATFORM); patch -p1 -i ../size-max.patch
 	cd $(PLATFORM); patch -p1 -i ../add-dl-as-private-lib.patch
+ifeq ($(OS),darwin_embedded)
+	cd $(PLATFORM); patch -p1 -i ../disable-hw-accel-aarch64.patch
```

Comment:
Why patch the buildsystem, instead of usinging the already existing configuration switch?

Reply:
I didn't realise there was a config switch. Will change that tomorrow.